content/agent/troubleshooting.md

```diff
@@ -34,6 +34,30 @@ To enable the full debug mode:
 
 4. Wait a few minutes to generate some logs. [Look here](/agent/#log-locations) to find the location of the logs.
 
+### Obtaining debug logs from the container agent
+
+This process covers agent v6 only, for agent v5 refer to [the dedicated documentation on how to collect more logs with the Datadog container agent v5](/agent/faq/agent-5-container-more-log)
+
+Set `DD_LOG_LEVEL=debug` then start your agent and [send a flare](#send-a-flare).
```

Comment:
can you detail it's an environment variable?

Reply:
👌